### PR TITLE
Added sane defaults for Audit.

### PR DESCRIPTION
**Pull request by DeathByTape**

### CODE REVIEW

File: elide-annotations/src/main/java/com/yahoo/elide/annotation/Audit.java [hunk 1]
```diff
@@ -46,33 +46,33 @@
...
     String path() default "";
 
     /**
      * Operation code.
      *
-     * @return the int
+     * @return the int (default = 1)
      */
-    int operation();
+    int operation() default 1;
```
Comment (aklish) [inline elide-annotations/src/main/java/com/yahoo/elide/annotation/Audit.java]: -1?

File: elide-annotations/src/main/java/com/yahoo/elide/annotation/Audit.java [hunk 2]
```diff
@@ -46,33 +46,33 @@
...
 
     /**
      * Logging string template.
-     * @return the string
+     * @return the string (default = "Generic log")
      */
-    String logStatement();
+    String logStatement() default "Generic log";
```
Comment (aklish) [inline elide-annotations/src/main/java/com/yahoo/elide/annotation/Audit.java]: empty string

**Comment by aklish** [comment]
:+1:

**Comment by yahoocla** [comment]
CLA is valid!

File: elide-annotations/src/main/java/com/yahoo/elide/annotation/Audit.java [hunk 3]
```diff
@@ -46,33 +46,33 @@
...
-    String logStatement();
+    String logStatement() default "";
 
     /**
      * Unified expression language expressions that will be evaluated and substituted into the logging template.
-     * @return the string [ ]
+     * @return the string [ ] (default = "")
      */
-    String [] logExpressions();
+    String [] logExpressions() default "";
```
Comment (clayreimann) [inline elide-annotations/src/main/java/com/yahoo/elide/annotation/Audit.java]: Unspace this to match the definition above.

**Pull request merged**